platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java

```diff
@@ -65,21 +75,35 @@ public void onRequestPermissionsResult(int requestCode,
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        // call main activity if extraction done
-        String flagFilePath = "/storage/emulated/0/Android/data/com.maq.xprize.CCI.hindi/files/.success.txt";
+        sharedPref = getSharedPreferences("ExpansionFile", MODE_PRIVATE);
+        String flagFilePath = "/storage/emulated/0/Android/data/" + getPackageName() + "/files/.success.txt";

```

Comment:
Use appropriate method to get this package dir path. Use wherever possible.

Reply:
Updated using getPackageName()